source/modules/MEI.header.xml

```diff
@@ -107,6 +107,62 @@
       </attDef>
     </attList>
   </classSpec>
+  <classSpec ident="att.perfRes" module="MEI.header" type="atts">
+    <desc>Attributes that define the characteristics and components of the performance resource.</desc>
+    <classes>
+      <memberOf key="att.perfRes.basic"/>
+      <memberOf key="att.transposition"/>
+    </classes>
+    <attList>
+      <attDef ident="solo" usage="opt">
+        <desc>Use this attribute to identify the performance resource as a soloist especially in an accompanied work, such as a concerto or vocal solo.</desc>
+        <datatype>
+          <rng:ref name="data.BOOLEAN"/>
+        </datatype>
+      </attDef>
+    </attList>
+  </classSpec>
+  <classSpec ident="att.perfRes.basic" module="MEI.header" type="atts">
+    <desc>Attributes that define the characteristics and components of the performance resource or a performance resource list.</desc>
+    <classes>
+      <memberOf key="att.adlibitum"/>
+    </classes>
+    <attList>
+      <attDef ident="count" usage="opt">
+        <desc>Indicates the number of performers.</desc>
+        <datatype>
+          <rng:data type="positiveInteger"/>
+        </datatype>
+      </attDef>
+    </attList>
+  </classSpec>
+  <classSpec ident="att.adlibitum" module="MEI.header" type="atts">
+    <desc>Attributes that define characteristics on an entity.</desc>
```

Comment:
Is an 'entity' the same as a 'performance resource'? If so we should make this consistent; if not we should define what an "entity" is.

Reply:
In my understanding every performance resource can be considered an entity (of the bibliographic universe) but not every entity is necessarily a performance resource. So from that perspective "entity" is a broader/more generic concept. However, I agree that we should consider to align our terminology across descriptions so that it's clear what is meant.

Reply:
Yes, maybe this is to general. I used this words because I remember that we discussed that this attribute (and its class) should be so generic that this can also be used in other contexts in the future (e.g. to encode an 'ad libitum' section in the `<music>` section).
Would it be better to to say more precisely something like "Attributes that characterize the way of use of a resource"?

Reply:
I wouldn't try to be too general; the application of the attributes in this class are specific to performance resources, as far as I am aware (that is, `adlib` will generally always apply to some sort of performer, regardless of the context in which it is used.)

So perhaps rather than trying to be generic in where it will be used, try to be specific in what it does. Something like: "Attributes that describe a performance resource as <hi rend="italic">ad libitum</hi> (optional)."

Reply:
Okay, sounds good! I'll integrate it.